images/external.md

```diff
@@ -18,6 +18,29 @@ such as [`jupyter/r-notebook`](https://jupyter-docker-stacks.readthedocs.io/en/l
 and [`jupyter/datascience-notebook`](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html#jupyter-datascience-notebook),
 so you can immediately run R on Jupyter.
 
+### [b-data/jupyterlab-r-docker-stack](https://github.com/b-data/jupyterlab-r-docker-stack)
+
+Multi-arch (`linux/amd64`, `linux/arm64/v8`) docker images:
+
+* [`registry.gitlab.b-data.ch/jupyterlab/r/base`](https://gitlab.b-data.ch/jupyterlab/r/base/container_registry)
+  * [`registry.gitlab.b-data.ch/jupyterlab/r/r-ver`](https://gitlab.b-data.ch/jupyterlab/r/r-ver/container_registry) (4.0.4 ≤ version < 4.2.0)
+* [`registry.gitlab.b-data.ch/jupyterlab/r/tidyverse`](https://gitlab.b-data.ch/jupyterlab/r/tidyverse/container_registry)
+* [`registry.gitlab.b-data.ch/jupyterlab/r/verse`](https://gitlab.b-data.ch/jupyterlab/r/verse/container_registry)
+* [`registry.gitlab.b-data.ch/jupyterlab/r/geospatial`](https://gitlab.b-data.ch/jupyterlab/r/geospatial/container_registry)
+
+Images considered stable for R versions ≥ 4.2.0. Differences to
+[The Rocker versioned stack](../index.md) and the
+[Jupyter Docker Stacks](#jupyter-docker-stacks):
+
+1. Multi-arch: `linux/amd64`, `linux/arm64/v8`  
+   → Since R v4.0.4 (2021-02-15)
+1. Base image: [Debian](https://hub.docker.com/_/debian) instead of
+   [Ubuntu](https://hub.docker.com/_/ubuntu)
+1. IDE: [code-server](https://github.com/coder/code-server) instead of
+   [RStudio](https://github.com/rstudio/rstudio)
+1. Just Python – no [Conda](https://github.com/conda/conda) /
+   [Mamba](https://github.com/mamba-org/mamba)
```

Comment:
Could you explain this more clearly?
Probably talking about the Python to install Jupyter?

Reply:
Maybe "distribution-only" or something like that?

Reply:
> Could you explain this more clearly?

The [Jupyter Docker Stacks](https://github.com/jupyter/docker-stacks) once used Conda to install [python] packages. Now it uses Mamba.

The [b-data/jupyterlab-r-docker-stack](https://github.com/b-data/jupyterlab-r-docker-stack) uses the distibutions python to install [python] packages.

> Probably talking about the Python to install Jupyter?

Yes, this also includes the installation of `jupyerhub`, `jupyterlab`, etc.

Reply:
I know it and I understand it, but I think it is possible that to someone reading this it may seem as if we are suddenly talking about Python.
Perhaps the gist is that installing (Python and R) packages only from PYPI and CRAN instead of using conda or mamba?

Reply:
I deleted the whole section to avoid misunderstanding.